ckanext/datagovuk/plugin.py

```diff
@@ -53,6 +54,12 @@ def package_types(self):
         # registers itself as the default (above).
         return []
 
+    # IRoutes
+
+    def after_map(self, map):
+        map.connect('harvest_index', '/harvest', action='index')
```

Comment:
In theory, the ckanext-harvest extension already has connected the route with a name.  I think this is fine though, it's more explicit, and isn't as hard to go and find.